pkg/cli/admin/release/info.go

```diff
@@ -1619,27 +1619,29 @@ func describeBugs(out, errOut io.Writer, diff *ReleaseDiff, dir string, format s
 			continue
 		}
 		for _, commit := range commits {
-			if len(commit.Bugs.Bugs) == 0 {
+			if len(commit.Refs.Refs) == 0 {
 				continue
 			}
-			for _, v := range commit.Bugs.Bugs {
-				if _, ok := bugIDs[generateBugKey(v.Source, v.ID)]; !ok {
-					// We are using generated bug key according to the source type
-					// to prevent possible clashes like BUG 111 and OCPBUGS-111
-					bugIDs[generateBugKey(v.Source, v.ID)] = v
+			for _, v := range commit.Refs.Refs {
+				// ignore jira refs to non bugs
+				if v.Source == Jira && !(strings.HasPrefix(v.ID, "OCPBUGS-") || strings.HasPrefix(v.ID, "OCPBUGSM-")) {
```

Comment:
Why we are ignoring non bug jira refs?

Reply:
this code is in the function `describeBugs` which is only called when using the `oc adm release info --bugs` argument, which per the description:

```
  --bugs='':
	Generate bug listings from the changelogs in the git repositories
	extracted to this path.
```

so i made it so this function filters out anything that isn't a bug.  i'll update the comment to explain this better.

Reply:
updated